sql/hive/src/main/scala/org/apache/spark/sql/hive/TableReader.scala

```diff
@@ -47,6 +48,7 @@ import org.apache.spark.sql.internal.SQLConf
 import org.apache.spark.unsafe.types.UTF8String
 import org.apache.spark.util.{SerializableConfiguration, Utils}
 
+
```

Comment:
nit: please remove the unnecessary change.

Reply:
done